examples/js-cdn-typescript/README.md

```diff
@@ -0,0 +1,49 @@
+# Web app with Javascript CDN
+
+This is a basic web application written in TypeScript that uses RudderTyper client library along with JS CDN script.
+
+## Steps to run the sample app
+
+1. Run `npm run setup` to install the required dependencies.
+2. Replace `writeKey` and `dataPlaneUrl` values in `./public/index.html` file.
```

Comment:
The placeholders for write key and data plane URL are different in the sample app.

Reply:
Addressed